projects/1.20/assets/medieval-origins-revival/medievalorigins/lang/zh_cn.json

```
@@ -0,0 +1,425 @@
+{
+  "origin.medievalorigins.arachnae.name": "阿拉克涅",
+  "origin.medievalorigins.arachnae.description": "阿拉克涅（名字来自罗马神话被变成蜘蛛的那个下面就都这样叫了）拥有人类的力量和蜘蛛的灵巧与狡诈。是真正可怕的猎手。",
+
+  "origin.medievalorigins.banshee.name": "女妖",
+  "origin.medievalorigins.banshee.description": "女妖是一种从痛苦中诞生的精灵，她们的哭声可以震碎玻璃，使人发疯。在神话中，它们的出现通常被认为是死亡的预兆。",
+
+  "origin.medievalorigins.incubus.name": "梦魇",
+  "origin.medievalorigins.incubus.description": "梦魇是黑暗力量的邪恶仆人。他们在自己的领域中力量无比强大，但失去灵魂的他们会在纯净的源头面前显得软弱无力。",
+
+  "origin.medievalorigins.dwarf.name": "矮人",
+  "origin.medievalorigins.dwarf.description": "矮人务实、强壮、骄傲。他们擅长采矿和地下生活，但在白天却很难适应。 Rock and stone（来自深岩银河的梗）, 兄弟。",

```

```suggestion
  "origin.medievalorigins.dwarf.description": "矮人务实、强壮、骄傲。他们擅长采矿，喜好地下生活，但很难适应日光。 挖到手软，赚到盆满，老兄。",
```
https://www.bilibili.com/read/cv14845013/